### DISCUSSION TITLE
предложения - новые методы и свойства для сервиса игрока

### DISCUSSION THREAD

**Discussion by justqstn** [Sokol-Games-GmbH/PixelCombats.PublicApi#89] [Ideas]
Index GetPosition { get; } // возвращает координаты игрока
void SetPosition(index) // перемещение игрока по координатам

еще предлагаю другой вариант: 
Index Position { get; set; } // текущие координаты игрока

**Comment by kkohno** [comment]
все верно, это добавим. отмечу пока
